X/X13as/build_tarballs.jl

```diff
@@ -0,0 +1,80 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "X13as"
+version = v"1.1.60"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://www2.census.gov/software/x-13arima-seats/x13as/unix-linux/program-archives/x13as_asciisrc-v1-1-b60.tar.gz", "2bd53953a1bdd238a128b89e95e8e5fc14f33aa4a9e4c7f0fc3fe7323b73131c", unpack_target="./x13as_asciisrc"),
+    ArchiveSource("https://www2.census.gov/software/x-13arima-seats/x13as/unix-linux/program-archives/x13as_htmlsrc-v1-1-b60.tar.gz", "642f6b6a969c5c311252ca83845ea391ab8c3d59840c4dc2508f9c86095c7757", unpack_target="./x13as_htmlsrc"),
+    ArchiveSource("https://www2.census.gov/software/x-13arima-seats/x13as/unix-linux/program-archives/x13as_ascii-v1-1-b60.tar.gz", "593e6b63024181c9550c281587bfeb28b2dcf1658f4495d42eb8933c6868ee36"),
+    ArchiveSource("https://www2.census.gov/software/x-13arima-seats/x13as/unix-linux/program-archives/x13as_html-v1-1-b60.tar.gz", "a9f55329373092c33ed5b1908094c0dc037cc05bf8789bd4f7156abcaa2200ed"),
```

Comment:
Any particular reason why these aren't independent packages?

Reply:
Yes. The first two are two source distributions for two versions of the same executable. They provide the same algorithms, the only difference is that one writes its output in ascii formatted tables, the other in html files. They are always both released at the same time by the US Census Bureau and have the same version numbers. I don't think this is a significant difference to justify maintaining two separate packages for them. 

The third and the fourth files are included here only for the pdf files with documentation in them. For reason unknown to me, the US Census Bureau doesn't include the pdf documentation files with the source distributions, it publishes them separately. 

I hope this answers your question. If you insist, I can separte the ascii and the html versions; otherwise I prefer to keep them together in one package. Please let me know.